statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -104,26 +73,76 @@ class MLEModel(Model):
 
     def __init__(self, endog, k_states, exog=None, dates=None, freq=None,
                  **kwargs):
-        # Set the default results class to be MLEResults
-        kwargs.setdefault('results_class', MLEResults)
+        # Initialize the model base
+        super(MLEModel, self).__init__(endog=endog, exog=exog,
+                                              dates=dates, freq=freq,
+                                              missing='none')
 
-        super(MLEModel, self).__init__(endog, k_states, exog, dates, freq,
-                                       **kwargs)
+        # Store kwargs to recreate model
+        self._init_kwargs = kwargs
```

Comment:
Good point, but it looks like the other tsa models don't do this. Possibly I will postpone this, then?